app/controllers/sites_controller.rb

```diff
@@ -106,6 +106,7 @@ def users
     return unless authorize_resource(@site, READ_SITE)
 
     @users = ComputedPolicy.authorized_users(Policy::Actions::READ_SITE, @site)
```

Comment:
We need to define which users we'll see here. We can either use this criteria, or list all users who belong to a role for this site. @nekron ?